### PR TITLE
[15.0][MIG]stock_lot_scrap: Migration to 15.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

File: stock_lot_scrap/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
+# Copyright 2021 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Scrap Production Lot",
+    "summary": "This module adds a button in Production Lot/Serial Number "
+    "view form to Scrap all products contained.",
+    "version": "15.0.1.0.1",
```
Comment (pedrobaeza) [inline stock_lot_scrap/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Comment by pedrobaeza** [comment]
Please squash the commit into the migration one.

File: stock_lot_scrap/models/stock_production_lot.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                limit=1,
+            )
+            .id
+        )
+        for quant in quants:
+            scrap = scrap_obj.create(
+                self._prepare_scrap_vals(quant, scrap_location_id),
+            )
+            scraps |= scrap
+        result = self.env.ref("stock.action_stock_scrap").read()[0]
```
Comment (rousseldenis) [inline stock_lot_scrap/models/stock_production_lot.py:98]:
```suggestion
        result = self.env["ir.actions.act_window"]._for_xml_id("stock.action_stock_scrap")
```

File: stock_lot_scrap/models/stock_production_lot.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                        "scrap_location",
+                        "=",
+                        True,
+                    ),
+                ],
+                limit=1,
+            )
+            .id
+        )
+        for quant in quants:
```
Comment (rousseldenis) [inline stock_lot_scrap/models/stock_production_lot.py:93]: Change this in order to use multi create.

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review

**Comment by pedrobaeza** [comment]
Cancelled due to Denis' comments

**Pull request closed without merging**